lineman/app/services/current_user.coffee

```diff
@@ -4,8 +4,15 @@ angular.module('loomioApp').factory 'CurrentUser', ($rootScope, Records) ->
     window.Loomio.seedRecords.users = [] unless window.Loomio.seedRecords.users?
     window.Loomio.seedRecords.users.push window.Loomio.seedRecords.current_user
     Records.import(window.Loomio.seedRecords)
+
+    currentUser =  Records.users.find(window.Loomio.currentUserId)
+
     Records.memberships.fetchMyMemberships().then ->
+      currentUser.updateFromJSON membershipsLoaded: true
       $rootScope.$broadcast 'currentUserMembershipsLoaded'
-    currentUser =  Records.users.find(window.Loomio.currentUserId)
+
+    Records.discussions.fetchInbox().then ->
+      currentUser.updateFromJSON inboxLoaded: true
```

Comment:
Yea, I think we should have an ApplicationState service or something rather than attaching this to the current user